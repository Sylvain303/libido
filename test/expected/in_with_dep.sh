#!/bin/bash
#
# libido: code=bash(three)
#
# libido: expand code
# expanded from: code => three,two,one
one() {
    echo "code for one"
}
two() {
    for i in $(seq 1 2)
    do
        echo "two:$(one)"
    done
}
three() {
    echo "code for three"
    two
}

three ici
