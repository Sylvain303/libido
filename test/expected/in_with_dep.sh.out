code for three
two:code for one
two:code for one
